A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_IRSInit.js

```diff
@@ -0,0 +1,143 @@
+class CDUIRSInit {
+    static ShowPage(mcdu, lon) {
+        mcdu.clearDisplay();
+        mcdu.setTitle('IRS INIT');
+        let checkAligned = SimVar.GetSimVarValue("L:A320_Neo_ADIRS_STATE", "Number");
+        let emptyIRSGpsString = "--°--.--/---°--.--"
+        let arrowupdwn = "↑↓"
+        let larrowupdwn = arrowupdwn
+        let rarrowupdwn = ""
+        if (lon) {
+            rarrowupdwn = arrowupdwn
+            larrowupdwn = ""
+        }
+        let IRS1Status;
+        let IRS2Status;
+        let IRS3Status;
```

Comment:
minor: variables should start with a lowercase letter to differentiate them easily